lib/SimpleSAML/XHTML/Template.php

```diff
@@ -155,6 +156,8 @@ public function __construct(Configuration $configuration, $template, $defaultDic
 
             $this->twig = $this->setupTwig();
         }
+
+        $this->charset = 'UTF-8';
```

Comment:
This is an upstream-issue.. Symfony is not initializing this var, even on their latest version